userguide/tutorials/what_is_kill_bill.adoc

```diff
@@ -2,26 +2,30 @@
 
 == Introduction
 
-Kill Bill is a mature open-source platform which powers the billing and payment infrastructure of some of the largest e-commerce websites.
+Kill Bill is an open-source programmable platform that enables you to build custom billing solutions. That's a mouthful, to say the least, but think of it as building blocks to implement your own billing workflows.
 
-The *subscription billing* features let you create recurring or usage based subscriptions (e.g. "gold plan subscription at $9.99/month") and manage upgrades and downgrades. The system will take care of generating invoices, managing credits and triggering payments. Netflix, Dropbox and The New York Times are examples of websites which sell such subscriptions.
+Similar to the Unix philosophy of composability over monolithic design, Kill Bill provides APIs to create various pricing models, manage subscriptions, generate invoices, integrate with various payment gateways, etc. Whether you are selling access to a digital service for $10 a month or selling a multi-tier cell-phone data plan, Kill Bill provides the primitives to do it so you don't have to start from scratch.
 
-The *payment* features let you connect with virtually any payment gateway and processor (Stripe, Braintree, PayPal, Adyen, CyberSource, etc.) to charge your customers (using credit or debit cards, ACH, wire transfers, Bitcoin, etc.) for their subscriptions or one-off payments (e.g. shopping cart-type websites like Amazon or Zappos).
+_Does Kill Bill support per CPU-hour billing?_ is the wrong kind of question to ask when evaluating Kill Bill. This would be similar to asking _Does Spring Boot offer endpoints to store user data?_: Spring Boot gives you all the tools you need to create endpoints which can take in user data and let you persist that data in a database, without having to worry about parsing HTTP headers or setting up database connection pooling, but it doesn't offer a user data endpoint. This is similar in the case of Kill Bill: while it doesn't understand _CPU hour billing_, you can create a usage-based plan in the catalog, which is billed based on the number of consumed _CPU-hour_ units.
```

Comment:
I think this is going to lose a lot of people who don't know what Spring Boot is (hand raised). It's really just a repetition of the previous paragraphs, so I recommend deleting it. ([Kill your darlings](https://www.masterclass.com/articles/what-does-it-mean-to-kill-your-darlings)!)

Reply:
Fair enough. That being said, _bis repetita placent_ 😁

Can you think of another metaphor we could use here, to drive home that point? This is an aspect often misunderstood about Kill Bill.